packages/asc-ui/src/components/InfoCard/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as InfoCardStyle } from './InfoCardStyle'
```

Comment:
Do we need to export `InfoCardStyle` here? Is it conventional to do so?

Reply:
We always do for now, so if a user wants to target this style specifically, he / she is able to, Although, I do think it's something we can perhaps change? Let's talk / think this through